src/Sylius/Behat/Context/Api/Admin/ManagingCatalogPromotionsContext.php

```diff
@@ -164,6 +164,7 @@ public function iWantToModifyACatalogPromotion(CatalogPromotionInterface $catalo
 
     /**
      * @When /^I add action that gives ("[^"]+") percentage discount$/
+     * @When /^I edit its action so that it reduces price by ("[^"]+")$/
```

Comment:
Are you sure, that this step edits the existing action and not adds the new one?